README.md

```diff
@@ -13,7 +13,7 @@ $ npm install @integromat/proto --save-dev
 ### A. Usage via module import
 
 ```ts
-import { /* ... */ } from '@integromat/proto';
+import {} from /* ... */ '@integromat/proto';
```

Comment:
This is a weird formatting mistake... Can we fix it somehow? 🤔

Reply:
I tested it manually and it's happening as well.
I can manually save it without formatting, but that doesn't resolve the issue. 
Or we can ignore the whole file which we didn't want to do

Reply:
And could we just update it to something like:
```suggestion
import { ... } from '@integromat/proto';
```